### PR TITLE
Two new translation Playwright tests created.

### PR DESCRIPTION
**Pull request by georgebid**
### Prerequisites

- [/ ] I have added steps to test this contribution in the description below

### Description

This PR contains 2 new playwright tests:
 1. Tests that the filtering on translation dictionary keys works as expected.
 2. A test to check if the icons update to a tick if a translation is added. 😄 

To test:
- Make sure to run `npm ci` & `npx playwright install`
- I would suggest adding test.only to the start of the test so you don't have to watch 20+ tests run.
- run` npm run ui` 
- check to see if the test passes

### CODE REVIEW

**Comment by georgebid** [comment]
@nul800sebastiaan Any idea what this Nightly build failure is/what could be causing it? I tested my tests I swear 😭

**Comment by emmagarland** [comment]
Thanks for the tests and test running steps @georgebid, I (or another of our team) will have a look soon!

I'll wait for the reply re the tests to confirm, 

Emma

**Comment by emmagarland** [comment]
(It doesn't look like that build failure is anything to do with the new test)

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts [hunk 1]
```diff
@@ -23,7 +26,53 @@ test.describe('Translation', () => {
     // Assert
     await umbracoUi.isSuccessNotificationVisible();
 
+  });
+  test('Create a 2nd dictionary key and filter the keys', async ({umbracoUi, umbracoApi, page}) => {
```
Comment (lauraneto) [inline tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts:30]: Looking at this test I think you are assuming that the test above has run before, and that is not something you can actually rely on.
Example: you can run tests in parallel, or even individually, so it's good practice to make all tests independent of each other.
So in this case I think you could add a cleanup step at the end of the first test, and then in this one create 2+ new dictionary entries and use those.

**Review by lauraneto — CHANGES_REQUESTED** [review]

File: tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts (hunk 1 above)
Comment (georgebid) [reply tests/Umbraco.Tests.AcceptanceTest/tests/DefaultConfig/Translation/translation.spec.ts:30]: Thanks for all of your suggestions @lauraneto! All make good sense to me (and not sure why I didn't think of some...) I will put in some updated when I get a chance 😄

**Comment by georgebid** [comment]
I'm gonna close this PR as it's pretty outdated, but I'm aiming to still resolve this soon 🤞

**Pull request closed without merging**